--- ../../../bullet-2.82-r2704/src/BulletDynamics/ConstraintSolver/btSolverBody.h	2014-04-28 23:04:54.000000000 -0700
+++ /Users/crydalch/Desktop/bullet-2.82-r2704/src/BulletDynamics/ConstraintSolver/btSolverBody.h	2013-10-22 02:26:08.000000000 -0700
@@ -38,13 +38,7 @@
 
 	}
 
-/* workaround for clang 3.4 ( == apple clang 5.1 ) issue, friction would fail with forced inlining */
-#if (defined(__clang__) && defined(__apple_build_version__) &&  (__clang_major__ == 5) && (__clang_minor__ == 1)) \
-|| (defined(__clang__) && !defined(__apple_build_version__) && (__clang_major__ == 3) && (__clang_minor__ == 4))
-    inline __attribute__ ((noinline)) btSimdScalar(float    fl)
-#else
-    SIMD_FORCE_INLINE   btSimdScalar(float  fl)
-#endif
+	SIMD_FORCE_INLINE	btSimdScalar(float	fl)
 	:m_vec128 (_mm_set1_ps(fl))
 	{
 	}
